__tests__/src/components/TicketsTable.test.tsx

```diff
@@ -97,4 +97,65 @@ describe('TicketsTable', () => {
     render(<TicketsTable items={[]} />);
     expect(screen.getByText('Urgent')).toBeInTheDocument();
   });
+
+  it('renders Urgent on table if a case is Urgent', () => {
+    render(<TicketsTable items={[]} />);
+    expect(screen.getByText('Urgent')).toBeInTheDocument();
+  });
+});
```

Comment:
I don't think this is working as intended. I think you'll need to make sure the Urgent text is in a row and not the header.

```typescript
it('renders Urgent on table if a case is Urgent', () => {
  const urgentItem = { ...items[0], urgent: true };
  render(<TicketsTable items={[urgentItem]} />);
  const urgentCell = screen.getByRole('cell', { name: 'Urgent' });
  expect(urgentCell).toBeInTheDocument();
});
```